attest/eventlog.go

```diff
@@ -224,10 +232,10 @@ var hashBySize = map[int]crypto.Hash{
 	crypto.SHA256.Size(): crypto.SHA256,
 }
 
-func extend(pcr, replay []byte, e rawEvent) ([]byte, Event, error) {
+func extend(pcr, replay []byte, e rawEvent) ([]byte, []byte, error) {
```

Comment:
since this is returning two values of the same type, they need to be named

```
(extension, digest []byte, err error)
```

Reply:
Done.